cmake/FindSpicy.cmake

```diff
@@ -141,12 +130,8 @@ function (spicy_print_summary)
             "\nFound Spicy:           ${HAVE_SPICY}")
 
     if (HAVE_SPICY)
-        message(
-            "\nVersion:               ${SPICY_VERSION} (${SPICY_VERSION_NUMBER})"
-            "\nPrefix:                ${SPICY_PREFIX}"
```

Comment:
This one would actually be useful to keep for debugging what Spicy installation is being used. We could just set it to some descriptive string when building from inside Zeek.

Reply:
I will keep this part as is.